### ISSUE TITLE
make tests: error code mismatch

### ISSUE THREAD

**Issue by HenryMiller1** [icecc/icecream#194]
When I run tests on ubuntu trusty 14.04 (skipping debug fission tests) they fail

```
Running: /usr/bin/gcc -Wall -Werror -x c++ -c plain -o /home/hm68841/workspace/icecream-upstream/ice-test/tests/results/plain.o
Command successful.

Running: /usr/bin/g++ -c nonexistent.cpp
Local run exit code mismatch (4 vs 1)
make[2]: *** [test-run] Error 2
make[2]: Leaving directory `/home/hm68841/workspace/icecream-upstream/ice-test/tests'
make[1]: *** [test-full] Error 2
make[1]: Leaving directory `/home/hm68841/workspace/icecream-upstream/ice-test/tests'
make: *** [test] Error 2
```

**Comment by HenryMiller1** [comment]
You can see that this is no just my machine:  The same issue exists on

https://travis-ci.org/HenryMiller1/icecream/builds/212256058

**Comment by johnmiked15** [comment]
This makes me think the environment that's getting created doesn't actually match the system. This test works on fine on Ubuntu yakkety. I think the same issue causing this to fail is causing the other failure with the gsplit-dwarf test.

**Comment by HenryMiller1** [comment]
fixed with #196
